packages/node-core/src/indexer/storeCache/cacheModel.ts

```diff
@@ -337,7 +343,11 @@ export class CachedModel<
           this.model.destroy({where: {id: Object.keys(removeRecords)} as any, transaction: tx}),
       ]);
     }
-
+    this.csvExporters.forEach((csvStore: CsvStoreService) => {
+      tx.afterCommit(async () => {
+        await csvStore.export(records);
+      });
```

Comment:
How does error handling work with this? If there is an error with export, does it give a good enough message to debug it?